linkml_runtime/utils/schemaview.py

```diff
@@ -631,8 +631,8 @@ def get_slot(self, slot_name: SLOT_NAME, imports=True, attributes=True, strict=F
             for c in self.all_classes(imports=imports).values():
                 if slot_name in c.attributes:
                     if slot is not None:
-                        # slot name is ambiguous: return a stub slot
-                        return SlotDefinition(slot_name)
+                        raise ValueError(f'Attribute "{slot_name}" is already defined in another class, please use a '
```

Comment:
would be great to start making more specific exception types in these packages - super useful for downstream programming. so idk like

```python
class AmbiguityError(ValueError):
    pass

class AmbiguousNameError(AmbiguityError):
    pass
```

and then one would both catch `ValueError`'s as well as having a more specific catch for that type of exception if someone wanted to (rather than needing to catch `ValueError` and then check the message text to know what kind of ValueError was raised)